dist/index.html

```diff
@@ -179,8 +177,8 @@ <h2 class="partner__title">Partners</h2>
     <footer class="footer lato_regular">
       <img class="footer__logo about-logo" src="./img/logo/logo-white.png" alt="">
```

Comment:
- [ ] Kindly add descriptive text to the `alt` attribute describing the image as this makes your page more accessible in case of image load failure.

Reply:
Fixed